website/docs/r/container_group.html.markdown

```diff
@@ -134,6 +134,10 @@ A `container` block supports:
 
 * `memory` - (Required) The required memory of the containers in GB. Changing this forces a new resource to be created.
 
+~> **Note:** Gpu resources are currently only supported in Linux containers.
```

Comment:
we tend to put `note` fields below the field they're talking about - could we move this below the line below?

Reply:
Done, thanks.